packages/cli/README-source.md

```diff
@@ -549,18 +549,33 @@ In cases where Zapier needs to show an example record to the user, but we are un
 
 ## Input Fields
 
-On each trigger, search, or create in the `operation` directive - you can provide an array of objects as fields under the `inputFields`. Input Fields are what your users would see in the main Zapier user interface. For example, you might have a "Create Contact" action with fields like "First name", "Last name", "Email", etc. These fields will be able to accept input from previous steps in a Zap, for example:
+On each trigger, search, or create in the `operation` directive, you can provide fields as an array of objects under `inputFields`. Input Fields are what your users see in Zapier when setting up your app's triggers and actions. For example, you might have a "Create Contact" action with fields like "First name", "Last name", "Email", etc. These fields will be able to accept input from the user, or from previous steps in a Zap. For example:
 
 ![gif of setting up an action field in Zap Editor](https://cdn.zappy.app/52721a3cb202446b7c298e303b710471.gif)
 
-You can find more details about setting action fields from a user perspective in [our help documentation](https://zapier.com/help/creating-zap/#set-up-action-template).
+You can find more details about setting action fields from a user perspective in [our help documentation](https://zapier.com/help/creating-zap/).
 
-Those fields have various options you can provide, here is a succinct example:
+Those fields have various options you can provide. Here is a brief example:
 
 ```js
 [insert-file:./snippets/fields.js]
 ```
 
+Notably, fields come in different types, which may look and act differently in the Zap editor. The default field display is a single-line input field.
```

Comment:
@eliangcs I actually used this phrasing intentionally, because things like integer fields also display as a single-line input field. That way I didn't have to describe that being the case for each field that appears that way, only for the ones that don't.

Reply:
Looks like I misinterpreted your sentence, so it's good as-is. Thanks!